### ISSUE TITLE
Ability to request removal of account 

### ISSUE THREAD

**Issue by mirajchokshi** [speak/issues#17]
Initially signed up via email but would like to sign up via Slack instead. Would be helpful to have a support@speak.io email or a request removal link for individuals who signed up via email. 

May also be useful for testing.

**Comment by tommoor** [comment]
:+1: totally agree, in this case I can remove your original gmail signup manually if you like.

**Comment by awsmsrc** [comment]
@mirajchokshi if you use the same email when you sign up with slack it will automatically link your speak account and slack account

**Comment by tcharde** [comment]
> "if you use the same email when you sign up with slack it will automatically link your speak account and slack account"

Hmm. That hasn't been the case for us. We all just downloaded Speak today, and so far no one else in my organization has been able to sign in to our existing Slack team using the email addresses that we used to create our Slack accounts.

**Comment by tommoor** [comment]
Hey @tcharde this was in reference to the case where an account with an email address already exists - signing in with slack will link the accounts together based on email address. Are you able to signin with Slack fine?

**Comment by tcharde** [comment]
>Hey @tcharde this was in reference to the case where an account with an email address already exists - signing in with slack will link the accounts together based on email address. Are you able to signin with Slack fine?

Sorry. Yes, we can all sign in to *Slack*. (We use it every day.) However: 

1. Only ***I*** have been able to sign in to *Speak* using our existing Slack team domain. 

2. One coworker *has* been able to sign in to Speak using the email that she used to sign up with Slack, but: A) Speak is showing a generic avatar (not her Slack avatar), and B) neither of us can see each other in Speak. 

3. Another coworker hasn't been able to sign in to Speak at all (it just keeps looping him through the process of entering our Slack team domain.) 

4. Although everyone received Speak invite codes from me, no one has seen a field / prompt for where to enter the codes after downloading, installing and attempting to login to Speak via Slack.

Again, sorry if this was the wrong place to post. Looks like we're having several different issues and I wasn't sure which to put in under.

**Comment by willbarrett** [comment]
Hey @tcharde - we added the ability to accept an invite after sign-in yesterday. It's in the Preferences area at the top of the screen, so your users should be able to accept those invites. If you'd like, we can also simply move all of the users you have to the correct organization on our side - just send a list of their email addresses to howdy@speak.io and we'll get it sorted out!

**Comment by tcharde** [comment]
Hi @willbarrett. I downloaded v0.3.0 and *can* see the "change team / invite code" in Preferences... But after installing the new version on everyone elses' systems, no one can get into the Preferences screen to enter the code (it's grayed out). I even tried removing Speak (and all associated files, via App Cleaner) and reinstalling, but no dice.

The weird part is that after choosing to sign in with Slack on Speak's initial sign-in/up screen, everyone can see our team listed as an option -- but once it's clicked, Speak just loop-reloads the initial screen again. 

I'll send you a list of email addresses for my team members that I've invited to Speak, and we'll see if that works. Thanks for helping troubleshoot this.
